libraries/chain/include/eosio/chain/hotstuff/hotstuff.hpp

```diff
@@ -123,14 +51,14 @@ namespace eosio::chain {
       bls_signature            _sig;
    };
 
-   // -------------------- quorum_certificate -------------------------------------------------------
+   // quorum_certificate
```

Comment:
I liked the previous separators better!

Reply:
Oh, the existing code base does not have this style of separators. Will not touch others.